stored_requests/backends/db_provider/mysql_dbprovider.go

```diff
@@ -22,8 +29,12 @@ func (provider *MySqlDbProvider) Config() config.DatabaseConnection {
 }
 
 func (provider *MySqlDbProvider) Open() error {
```

Comment:
Just confirming, I assume you've tested this with a remote database instance right? We haven't done that on our end for MySQL; we only tested it locally.

The documentation does appear to indicate that only the `tls` query param is needed.

Reply:
Yes, I've tested it with a remote database hosted at DigitalOcean and with a local database running in a docker container.